tests/k8s/client-w-conf.yml

```diff
@@ -5,12 +5,12 @@ metadata:
 spec:
   selector:
     matchLabels:
-       run: client
+       name: client
   replicas: 5
   template:
     metadata:
       labels:
-        run : client
```

Comment:
don't we need a similar change to the other `yaml` files ? (bootstrap, poet, single pod)

Reply:
yes we need.
I changed this also in single-client-w-conf.yml